rusoto/core/src/signature.rs

```diff
@@ -310,6 +341,7 @@ fn canonical_headers(headers: &BTreeMap<String, Vec<Vec<u8>>>) -> String {
     canonical
 }
 
+/// Converts values into standardised form.
```

Comment:
http://docs.aws.amazon.com/general/latest/gr/sigv4-create-canonical-request.html

Would still be the place I'd link to for this (yes it's the same as above). :smile: